.github/workflows/clang-format.yml

```
@@ -17,7 +17,7 @@
 #
 
 ---
-name: BE Code Formatter
+name: BE Code Checks
 on:
   pull_request:
```

Use `pull_request_target` to enable comment feature.
```suggestion
  pull_request_target
```